components/policy-mgt/org.wso2.carbon.policy.mgt.core/src/main/java/org/wso2/carbon/policy/mgt/core/mgt/impl/MonitoringManagerImpl.java

```diff
@@ -247,13 +251,25 @@ public void addMonitoringOperation(List<Device> devices) throws PolicyCompliance
             String msg = "Error occurred from monitoring dao.";
             log.error(msg, e);
             throw new PolicyComplianceException(msg, e);
+        } catch (OperationManagementException e) {
+            String msg = "Error occurred while adding monitoring operation to devices";
```

Comment:
Removed in commit [1].
[1] https://github.com/wso2/carbon-device-mgt/commit/cdbdc86f42b350c24e9d059ff7c23e1f09242df4